### PR TITLE
Enabled cdi40 for faces40 automatically

### LINKED ISSUE #22187: Faces 4.0: Automatically enable CDI for Faces 4.0 now that ManagedBeans are gone

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #22187

### CODE REVIEW

**Comment by pnicolucci** [comment]
#build #spawn.fullfat.buckets=io.openliberty.org.apache.myfaces.4.0_fat,com.ibm.ws.jsf.2.3_fat,com.ibm.ws.jsf.2.2_fat

**Comment by pnicolucci** [comment]
#build #spawn.fullfat.buckets=io.openliberty.org.apache.myfaces.4.0_fat,com.ibm.ws.jsf.2.3_fat,com.ibm.ws.jsf.2.2_fat

**Review by isaacrivriv — DISMISSED** [review]
Changes look good, thanks for getting this in!

**Comment by pnicolucci** [comment]
#build #spawn.fullfat.buckets=io.openliberty.org.apache.myfaces.4.0_fat,com.ibm.ws.jsf.2.3_fat,com.ibm.ws.jsf.2.2_fat

**Review by volosied — DISMISSED** [review]
LGTM! Thanks :)

**Comment by pnicolucci** [comment]
#libby

**Comment by pnicolucci** [comment]
#libby #build

**Comment by pnicolucci** [comment]
#libby #build

File: dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/JSF22BeanValidationTests.java [hunk 1]
```diff
@@ -26,7 +26,6 @@
 import com.gargoylesoftware.htmlunit.html.HtmlPage;
 import com.gargoylesoftware.htmlunit.html.HtmlTextInput;
 import com.ibm.websphere.simplicity.ShrinkHelper;
-import com.ibm.websphere.simplicity.config.ServerConfiguration;
```
Comment (tevans78) [inline dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/JSF22BeanValidationTests.java:29]: Please remember to update the copyright date in the header of any file which has functionally changed (any change except whitespace or formatting).

Comment (pnicolucci) [reply dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/JSF22BeanValidationTests.java:29]: Work on these files was in 2022, they already have a 2022 date in the header. I had to rebase and do a force push so it was a bit confusing. We are all set here.

**Comment by pnicolucci** [comment]
#libby #build

**Comment by pnicolucci** [comment]
The only failure in the personal build is being fixed by https://github.com/OpenLiberty/open-liberty/pull/23864 and is not related to this PR.

**Review by isaacrivriv — APPROVED** [review]

**Pull request merged**